neuro_san/service/mcp/processors/mcp_tools_processor.py

```diff
@@ -222,3 +256,8 @@ def _get_chat_input_request(self, input_text: str) -> Dict[str, Any]:
                 "chat_filter_type": "MAXIMAL"
             }
         }
+        if chat_context is not None:
+            request_dict["chat_context"] = chat_context
+        if sly_data is not None:
+            request_dict["sly_data"] = sly_data
+        return request_dict
```

Comment:
Construct what we will send to internal neuro-san chat request using chat_context and sly_data if they are in tool call parameters.

Reply:
No sly_data back to caller's chat stream